crafters/nlp/PDFExtractor/tests/test_pdfextractor.py

```diff
@@ -0,0 +1,25 @@
+from .. import PDFTextExtractor
+
+def create_pdf():
+    from reportlab.pdfgen.canvas import Canvas
```

Comment:
if you use the page interface, make sure you are creating a pdf with multiple pages

Reply:
Yeah the example one has 2 pages with one image to test that